apps/console/src/features/connections/components/edit/settings/attribute-settings.tsx

```diff
@@ -356,6 +358,27 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
         <EmphasizedSegment padded="very">
             <Grid className="attributes-settings">
                 <div className="form-container with-max-width">
+                    {
+                        !UIConfig?.isCustomClaimMappingEnabled && (
+                            <Grid.Row columns={ 1 }>
+                                <Grid.Column>
+                                    <Message
+                                        type="warning"
+                                        content={
+                                            (
+                                                <Trans>
+                                                    <strong>Custom Attribute Mapping</strong> is disabled in
+                                                    your configuration. This might affect certain flows in the
+                                                    application. Proceed with caution.
```

Comment:
```suggestion
                                                    <strong>Custom Attribute Mapping</strong> is disabled in
                                                    your system configuration. This might affect certain flows 
                                                    related to the connection. Proceed with caution.
```

Reply:
Fixed with af68c2afdd1657f85b07719938222114b70cc090